plugins/foods.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, name, unit, *commands, file=None):
     BasicFood("kebab", "a kebab"),
     BasicFood("cake", "a cake"),
     BasicFood("hotchoc", "hotchoc"),
+    BasicFood("squid", "squid"),
```

Comment:
```suggestion
    BasicFood("squid", "some squid"),
```
So it will say ".squid - Serves <user> some squid", "I can't serve some squid to that user", etc